CONTRIBUTING.md

```diff
@@ -138,6 +138,19 @@ Because SQL within JSON or YAML will not generally be syntax highlighted, indent
 * One space before and after = etc
 * Name SQL subqueries after the layer name (but use underscores)
 
+## Map Icon Guidelines
+
+Design Guidelines
+* SVG only
+* flat meaning a single colour, no gradients, no outlines
+* common canvas size, which is usually 14x14 px
+* single point of view and avoid use of perspective where possible
```

Comment:
Also, what "single point of view" means in context of making icons? This one is unclear for me what it means for graphics that are without perspective.